config/src/main/java/com/alibaba/nacos/config/server/auth/EmbeddedUserPersistServiceImpl.java

```diff
@@ -117,4 +118,11 @@ public User findUserByUsername(String username) {
         }
         return pageInfo;
     }
+
+    @Override
+    public List<User> findUserLikeUsername(String username) {
+        String sql = "SELECT username FROM users WHERE username like '%' ? '%'";
```

Comment:
Please use prepareStatement and `?` SQL to resolve SQL injection risk.

Just like `findUserByUsername` method